cfgov/jinja2/v1/rural-or-underserved/index.html

```diff
@@ -23,6 +23,7 @@
     {{ super() }}
     <script>
       if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
+        window.mapboxAccessToken = '{{ mapbox_access_token }}'
```

Comment:
What do you think about prefixing these global variables with `cfpb`, so if the window object is examined in inspector, the global variables specific to CFPB apps are aggregated together and found in the same spot.

And on that note, should this be the same as https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/find-a-housing-counselor/js/common.js#L7 ? 
```js
const mapboxAccessToken = window.cfpbHudSettings.mapbox_access_token;
```